doc/source/bioformats.rst

```diff
@@ -0,0 +1,14 @@
+Bioformats
+==========
+
+The ``Bioformats`` reader interfaces with Bio-Formats, an open-source Java
+library for reading and writing multidimensional image data, especially from
+file formats used in microscopy.
+
+See the `list of supported formats <https://www.openmicroscopy.org/site/support/bio-formats5.1/supported-formats.html>`_.
+
+Depedencies
+-----------
+To interface with the java library, we use `JPype <https://github.com/originell/jpype>`_, which allows fast and easy access to all java functions. JRE or JDK are not required.
+
```

Comment:
For installation with ``pip``, make sure to use ``pip install jpype1`` to obtain the correct version (0.6.0 or later).